src/app/client/src/app/app.component.ts

```diff
@@ -313,11 +317,38 @@ export class AppComponent implements OnInit, OnDestroy {
     //       this.isStepperEnabled = true;
     //     }
     //     else { this.isPopupEnabled = true; }
-    //   }, error => { this.isPopupEnabled = true; });
+    //   }, error => { this.isPopupEnabled = true; });;
     this.isPopupEnabled = true;
   }
+
+  getOnboardingSkipStatus(){
+    /**
+      * @description - This method enables/disables the onboarding popups based on the isvisible values returned from the form config request
+    */
+    const formReadInputParams = {
+      formType: 'onboardingPopupVisibility',
+      formAction: 'onboarding',
+      contentType: "global",
+      component: "portal"
+    };
+    this.formService.getFormConfig(formReadInputParams).subscribe(
+      (formResponsedata) => {
+        if (formResponsedata) {
+          this.popupControlService.setOnboardingData(formResponsedata);
+          this.isOnboardingEnabled= formResponsedata.onboardingPopups? formResponsedata.onboardingPopups.isVisible : true;
+          this.isFWSelectionEnabled = formResponsedata.frameworkPopup? formResponsedata.frameworkPopup.isVisible : true;
+          this.isUserTypeEnabled =formResponsedata.userTypePopup? formResponsedata.userTypePopup.isVisible: true;
+          if(this.isOnboardingEnabled === false || this.isFWSelectionEnabled === false){
```

Comment:
@mithun30052001  refactor this line of code if(this.isOnboardingEnabled === false || this.isFWSelectionEnabled === false){

Reply:
Refactor done @princegupta1131